content/courses/program-security/pda-sharing.md

```diff
@@ -189,161 +184,149 @@ pub mod pda_sharing_recommended {
             ctx.accounts.pool.withdraw_destination.as_ref(),
             &[ctx.accounts.pool.bump],
         ];
-        token::transfer(get_transfer_ctx(&ctx.accounts).with_signer(&[seeds]), amount)
+        token::transfer(ctx.accounts.transfer_ctx().with_signer(&[seeds]), amount)
     }
 }
 
 #[derive(Accounts)]
 pub struct WithdrawTokens<'info> {
     #[account(
-        seeds = [withdraw_destination.key().as_ref()],
-        bump = pool.bump,
-        has_one = vault,
-        has_one = withdraw_destination,
-    )]
+				has_one = vault,
+				has_one = withdraw_destination,
+				seeds = [withdraw_destination.key().as_ref()],
+				bump = pool.bump,
+		)]
```

Comment:
Check your editor settings, pretty sure you're using tabs. Rustfmt uses 4 spaces.